kf/bpf.go

```diff
@@ -243,6 +253,19 @@ func (b *BPF) Stop(ifaceName, direction string, chain bool) error {
 	stats.SetWithVersion(0.0, stats.NFRunning, b.Program.Name, b.Program.Version, direction, ifaceName)
 
 	if len(b.Program.CmdStop) < 1 {
+		// Program ref handle - loaded from l3afd
+		if b.CollectionRef != nil {
```

Comment:
I think this should be above/outside the if block.  If there is a CollectionRef defined we should unload it as I believe this means we loaded the eBPF program directly and didn't run any external binary to do this.  In this case wouldn't b.Program.CmdStop be irrelevant?